src/sky/pro/average/Employee.java

```diff
@@ -0,0 +1,50 @@
+package sky.pro.average;
+
+public class Employee {
+    final private String fullName;
```

Comment:
```suggestion
    private final String fullName;
```
это более общепринятый порядок объявления модификаторов